PowerShellGet/PSModule.psm1

```diff
@@ -4931,7 +4941,7 @@ function New-ScriptFileInfo
 
         [Parameter()]
         [ValidateNotNullOrEmpty()]
-        [Version]
+        [String]

```

Comment:
>[String] [](start = 8, length = 8)

Minor: For string type parameter, the best practice is to use lowercase [string] instead of [String]. Take a look at above $Path param type.